cns/configuration/configuration.go

```diff
@@ -120,11 +124,14 @@ func setManagedSettingDefaults(managedSettings *ManagedSettings) {
 	}
 }
 
-// Set Default values of CNS config if not specified
+// SetCNSConfigDefaults set default values of CNS config if not specified
 func SetCNSConfigDefaults(config *CNSConfig) {
 	setTelemetrySettingDefaults(&config.TelemetrySettings)
 	setManagedSettingDefaults(&config.ManagedSettings)
 	if config.ChannelMode == "" {
 		config.ChannelMode = cns.Direct
 	}
+	config.SyncHostNCVersionIntervalMilliSec = 1000
+	config.SyncHostNCTimeoutMilliSec = 500
+	config.ForceMarkIPAvailableTimeRange = 5000
```

Comment:
What does ForceMarkIPAvailableTimeRange  represent?

Reply:
It represents the AKS fall back logic when NMAgent is done,

Reply:
Discuessed offline, moved this part.